listings/applications/erc20/src/erc20_streaming.cairo

```diff
@@ -60,11 +63,13 @@ pub mod erc20_streaming {
             ref self: ContractState,
             to: ContractAddress,
             total_amount: felt252,
-            start_time: u64,
             end_time: u64
         ) {
             assert(total_amount != felt252::zero(), Errors::STREAM_AMOUNT_ZERO);
             let caller = get_caller_address();
+            let start_time = get_block_timestamp(); // Use block timestamp for start time
```

Comment:
I think it's still ok to keep `start_time` as argument, it can be useful to start a stream later in time.
Just add the assert `start_time >= get_block_timestamp()`